### PR TITLE
s1_reader: add default pol = vv

### PR DESCRIPTION
**Pull request by yunjunz**
This PR adds the following minor changes:

+ move `requirements.txt` file to the root level directory, for a shorter fie path

+ remove `swath` from orbit-related function names, for a shorter and more intuitive name, as orbit file does not depend on swath num.

+ `sentinel1_reader.burst_from_zip()`: add default `pol = "vv"`

### CODE REVIEW

**Review by hfattahi — APPROVED** [review]
LGTM!

**Review by LiangJYu — COMMENTED** [review]
Looks good. Just a minor comment on import order.

File: src/sentinel1_reader/sentinel1_burst_slc.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-from dataclasses import dataclass
 import os
+import datetime
 import tempfile
-
-import isce3
 import numpy as np
+from dataclasses import dataclass
 from osgeo import gdal
+import isce3
```
[the remark this answers is not in the document]
Comment (yunjunz) [reply src/sentinel1_reader/sentinel1_burst_slc.py]: Thank you for the explanation. I have adjusted to your suggested order.

**Review by LiangJYu — APPROVED** [review]
LGTM

**Pull request merged**